### PR TITLE
adding my animation

### PR DESCRIPTION
**Pull request by SEN-RAD**
Adding to the project, just for fun :) Please, add me to this list. @NunoCPNP

### CODE REVIEW

**Comment by SEN-RAD** [comment]
My fork was outdated. I'm resolving the conflict with the master branch. Hope OK now.

File: include.js [hunk 1]
```diff
@@ -6275,6 +6276,13 @@ let cards = [
     author: 'Katana',
     githubLink: 'https://github.com/katanaji'
   }
```
Comment (sophiabrandt) [inline include.js:6278]: Please add the missing comma between curly brackets to avoid syntax errors.

```js
},
{
```

**Review by sophiabrandt — APPROVED** [review]
Please fix syntax errors

**Comment by SEN-RAD** [comment]
My work has been added already. Closing this pull request.

**Pull request closed without merging**